controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -559,6 +559,17 @@ func (a *ExistingInfraMachineReconciler) update(ctx context.Context, c *existing
 		return a.kubeadmUpOrDowngrade(ctx, c, machine, eim, node, installer, version, planJSON, recipe.Worker)
 	}
 
+	isOriginal, err := a.isOriginalMaster(ctx, node)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Is original control plane node: ", isOriginal)
+	log.Info("ControlPlaneEndpoint: ", c.Spec.ControlPlaneEndpoint)
+	if isOriginal && c.Spec.ControlPlaneEndpoint == "" {
+		return errors.New("cannot perform update of original control plane node without a control plane load balacer")
```

Comment:
Will errors at this point put it back in the queue and try to reconcile it again?

Reply:
Upgrades will still work w/out an LB I guess, but updating config / anything that requires a repaving will not?

Reply:
True, this was added because there was no way to create a node without generating a diff in the node plan at the `selinux:permissive` step right after. Repaving the nodes for when there's a diff, and not having the cp LB, resulted in dropping the connection to kubectl every time.
Returning `nil` and logging the error message should also work, and avoid the stack trace from `zapLogger`, so this is an option!